src/main/java/com/exasol/adapter/RequestDispatcher.java

```diff
@@ -79,8 +80,10 @@ private String processRequest(final AdapterRequest request, final VirtualSchemaA
         case PUSHDOWN:
             return dispatchPushDownRequestToAdapter(request, adapter, metadata);
         default:
-            throw new AdapterException("The request dispatcher encountered a request type \"" + type.toString()
-                    + "\" which it does not recognize. Please create an issue ticket quoting this error message.");
+            throw new AdapterException(ExaError.messageBuilder("E-VS-COM-JAVA-30")
+                    .message("The request dispatcher encountered a request type {{type}} which it does not recognize.")
+                    .mitigation("Please create an issue ticket quoting this error message.")
```

Comment:
replace using ticketMitigation(). That allows us to change the message in the future